### PR TITLE
Websocket usage

### PR DESCRIPTION
**Pull request by imscaradh**
* Global notifier construct -> Can be used for global modal notifications
* First draft of live data streaming with Chart.js
* Typo fixes
* Environment configuration modifications to use native websocket URL

### CODE REVIEW

**Comment by curtys** [comment]
Can you adapt the mock api please?

File: src/app.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ export class App {
     }
 
     public attached(owningView, myView): void {
-        const ws = new WebSocket(`ws://${this.config.get('api.host')}`);
+        const ws = new WebSocket(`ws://${this.config.get('api.host')}/notifications`);
```
[the remark this answers is not in the document]
Comment (imscaradh) [reply src/app.ts]: @imscaradh I didn't set up SSL yet but locally it should work without this step. But I think we have either to switch from Socket.IO to ws (https://www.npmjs.com/package/ws) or vice versa. Wdyt?

**Comment by curtys** [comment]
> The implementation still doesn't work with the mock api :(

I changed the mock implementation and removed socket.io (apparently the are not compatible).
But you have to send the data in the form client expect, did not do that.

File: src/app.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (curtys) [reply src/app.ts]: I removed socket.io. It is not compatible with other clients.
As long as we don't have SSL this should be fine.

**Pull request merged**